.editorconfig

```diff
@@ -0,0 +1,22 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = crlf
```

Comment:
Is LF not better here?

Reply:
Can we please stick with Unix line endings here?